lib/cadet/assessments/assessments.ex

```diff
@@ -753,6 +809,69 @@ defmodule Cadet.Assessments do
     |> Repo.transaction()
   end
 
+  def update_contest_voting_submission_status(submission = %Submission{}, question = %Question{}) do
```

Comment:
This function assumes that the voting assessment has only one question too. Can we get rid of this assumption?